app/Http/Services/WithdrawalService.php

```
@@ -0,0 +1,97 @@
+    public function __construct(RequestWithdrawal $model)
+    {
+        $this->model = $model;
+    }
+    /**
+     * Withdraw items from warehouse using FIFO logic.
+     *
+     */
+    public function withdrawItemsFromWarehouse($requestWithdrawalItems)
+    {
+        DB::transaction(function () use ($requestWithdrawalItems) {
+            Log::info("Processing withdrawal items [{$requestWithdrawalItems}]");
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'withdrawItemsFromWarehouse'. (undefined)

(StaticAccess)